README.md

```diff
@@ -17,9 +17,12 @@ It is able to create rectilinear trajectories on complicated surfaces (3D surfac
 Dependencies
 ------------
 
-- ROS indigo version (with the [Fanuc package](https://github.com/ros-industrial/fanuc))
-- VTK 7.0 or later, you need ([this patch](https://gitlab.kitware.com/vtk/vtk/merge_requests/213)), using the `master` branch is fine.
-- The [Point Cloud Library](https://github.com/PointCloudLibrary/pcl) version 1.8 or trunk
+- [ROS](http::wiki.ros.org/ROS/Installation) (Robot Operating System)
+- [`insdustrial-core`](http://wiki.ros.org/industrial_core)
+- [`fanuc`](https://github.com/InstitutMaupertuis/fanuc) these packages were modified by Institut Maupertuis
+- [`fanuc experimental`](https://github.com/InstitutMaupertuis/fanuc_experimental) these packages were modified by Institut Maupertuis
```

Comment:
For `fanuc` and `fanuc_experimental`, add a warning like this:
https://github.com/InstitutMaupertuis/institut_maupertuis_robots_descriptions/blob/indigo-devel/README.md#dependencies